src/freerct.cpp

```diff
@@ -126,13 +127,12 @@ int freerct_main(int argc, char **argv)
 
 	InitMouseModes();
 
-	StartNewGame();
+	/// \todo Allow for loading directly from a saved game.
+	_game_control.NewGame();
 
 	/* Loops until told not to. */
 	_video.MainLoop();
 
```

Comment:
Stuff is out of balance here, _game_control starts a new game outside _Video, but it doesn't close down here.

Code becomes easier to understand if you keep actions that come in pairs at the same level. Please consider either moving _game_control.NewGame() inside _video, or having the close action here after _video as well.